### PR TITLE
Update README and Makefile

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +17,18 @@ Options:
...
+sudo make install
+```
+
+## Removal
+
+```
+sudo rm /usr/bin/local/tldr
+```
+
+
```
Comment (CleanMachine1) [inline README.md]:
```suggestion
```

**Comment by CleanMachine1** [comment]
Feel free to completely ignore this and close if you feel it is wrong.

1 suggestion I can make is adding a : after the headings if you agree.

Anyway, hope you find my commits slightly useful.

File: Makefile [hunk 2]
```diff
@@ -1,2 +1,5 @@
 build:
-	g++ src/*.cpp -std=c++17 -lzip -lcurl -o tldr
\ No newline at end of file
+	g++ src/*.cpp -std=c++17 -lzip -lcurl -o tldr
+
+install: 
+	cp ./tldr /usr/loca/bin/
```
Comment (marchersimon) [inline Makefile]:
```suggestion
	cp ./tldr /usr/local/bin/
```
Why not `/usr/bin/`? I don't really get the difference.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# A TL;DR client made in C++
```
Comment (marchersimon) [inline README.md]:
```suggestion
# The all-in-one TL;DR client for users and contributors
```
I actually prefer my current descripion.

File: README.md [hunk 4]
```diff
@@ -13,3 +17,16 @@ Options:
       --raw       Display the raw Markdown page without formatting
   -h, --help:     Display this help
 ```
+
+## Installation 
+
+```
+make build 
+sudo make install
```
Comment (marchersimon) [inline README.md:25]: If we woud write 
```
install: build
 ...
```
in the Makefile we could skip the `make build` step, right?

**Review by marchersimon — COMMENTED** [review]
Great, thanks a lot!
Did you also test it and maybe encountered any bugs?

File: Makefile (hunk 2 above)
Comment (CleanMachine1) [reply Makefile]: I don't see the problem with /usr/bin, if you prefer it, I will change to it.

File: README.md (hunk 4 above)
Comment (CleanMachine1) [reply README.md:25]: Then we will have sudo for g++, some people will prefer to not run sudo with that. Better to make it 2 commands.

File: README.md [hunk 5]
```diff
@@ -13,3 +17,16 @@ Options:
       --raw       Display the raw Markdown page without formatting
   -h, --help:     Display this help
 ```
+
+## Installation 
+
+```
```
Comment (CleanMachine1) [inline README.md]: Would we like to add shell to that top line to signify it is commands to be run

File: README.md (hunk 4 above)
Comment (marchersimon) [reply README.md:25]: Ah, okay.

File: README.md (hunk 5 above)
Comment (marchersimon) [reply README.md]: Sure 👍🏽

File: README.md [hunk 6]
```diff
@@ -13,3 +17,16 @@ Options:
...
+make build 
+sudo make install
+```
```
Comment (marchersimon) [inline README.md:26]:
````suggestion
```shell
make build 
sudo make install
```
````

**Pull request merged**